air/src/execution_step/execution_context/context.rs

```diff
@@ -96,9 +87,29 @@ impl<'i> ExecutionCtx<'i> {
     }
 }
 
+use serde::Deserialize;
+use serde::Serialize;
 use std::fmt::Display;
 use std::fmt::Formatter;
 
+/// It reflects RunParameters structure due to limitation of the marine macro to support Rc.
+#[derive(Debug, Default, Clone, Eq, PartialEq, Serialize, Deserialize)]
+pub(crate) struct RcRunParameters {
```

Comment:
Hmm, don't know tbh, I think that `Rc` prefix makes more sense then `Light` suffix, because one could interpret `Light` differently. Let's stick with `Rc` until we come out with a better idea.